e2e/e2e.rs

```diff
@@ -0,0 +1,3 @@
+#[cfg(test)]
+mod tests;
+pub extern crate chrono;
```

Comment:
Ah, yes, we shouldn't have sticked to an old input
Apparently its not necessary, removed. Once `prometheus_query` would be fixed it would be enabled there